tmp.py

```diff
@@ -0,0 +1,74 @@
+from textx import metamodel_from_str
```

Comment:
This file is here by mistake I suppose.

But, it seems it would be a very nice addition to the examples. :)

Reply:
Removed file.
This will be for a later pull request (I would like to add the ability to add lists of object_processors) - then we can have multiple validators (logical rules) for one grammar rule... Details are not finished.

Reply:
for this PR: OK